tsconfig.json

```diff
@@ -9,5 +9,6 @@
     "public/*"
   ],
   "compilerOptions": {
+    "allowJs": true
```

Comment:
what do you need this for?

Reply:
This was needed to import routify generated file which is in javascript.